system-test/test.sh

```diff
@@ -6,6 +6,18 @@ retry() {
   "${@}" || "${@}" || "${@}" || return 1
 }
 
+function wait_only() {
+  if [ -f /bin/busybox ]; then
+    timeout -t "${@}"
+  else
+    timeout "${@}"
+  fi
+}
+
+npm_install() {
```

Comment:
This name is too similar to the ```npm install``` command. Naming it ```timed_npm_install``` may make it less confusing and convey its intent. 

It is a bit subjective, feel free to resolve this without any further action.

Reply:
This is a fair point, but I'm going to mark this as resolve, because this is consistent with the name used in cloud-profiler-nodejs, and I know I'm unlikely to go back and update the name there.